fusion-cli/entries/client-entry.js

```diff
@@ -16,6 +16,9 @@ function reload() {
   const main = require('__FUSION_ENTRY_PATH__'); // eslint-disable-line
   const initialize = main.default || main;
   Promise.resolve(initialize()).then(app => {
+    if (!app) {
```

Comment:
Should we check if this is an instance of BaseApp from fusion-core?

Reply:
That works!